pages/settings/index.vue

```diff
@@ -0,0 +1,243 @@
+        }
+      ],
+      adminItems: [
+        {
+          icon: 'mdi-server',
+          name: this.$t('settingsSections.server.name'),
+          description: this.$t('settingsSections.server.description')
+        },
+        {
+          icon: 'mdi-devices',
+          name: this.$t('settingsSections.devices.name'),
+          description: this.$t('settingsSections.devices.description')
+        },
+        {
+          icon: 'mdi-account-multiple',
+          name: this.$t('settingsSections.users.name'),
+          description: this.$t('settingsSections.users.description')
+        },
+        {
+          icon: 'mdi-library-shelves',
+          name: this.$t('settingsSections.libraries.name'),
+          description: this.$t('settingsSections.libraries.description')
+        },
+        {
+          icon: 'mdi-play-network',
+          name: this.$t('settingsSections.transcodingAndStreaming.name'),
+          description: this.$t(
+            'settingsSections.transcodingAndStreaming.description'
+          )
+        },
+        {
+          icon: 'mdi-dlna',
+          name: this.$t('settingsSections.dlna.name'),
+          description: this.$t('settingsSections.dlna.description')
+        },
+        {
+          icon: 'mdi-television-classic',
+          name: this.$t('settingsSections.liveTvAndDvr.name'),
+          description: this.$t('settingsSections.liveTvAndDvr.description')
+        },
+        {
+          icon: 'mdi-network',
+          name: this.$t('settingsSections.networking.name'),
+          description: this.$t('settingsSections.networking.description')
+        },
+        {
+          icon: 'mdi-puzzle',
+          name: this.$t('settingsSections.plugins.name'),
+          description: this.$t('settingsSections.plugins.description')
+        },
+        {
+          icon: 'mdi-key-chain',
+          name: this.$t('settingsSections.apiKeys.name'),
+          description: this.$t('settingsSections.apiKeys.description')
+        },
+        {
+          icon: 'mdi-calendar-clock',
+          name: this.$t('settingsSections.scheduledTasks.name'),
+          description: this.$t('settingsSections.scheduledTasks.description')
+        },
+        {
+          icon: 'mdi-bell',
+          name: this.$t('settingsSections.notifications.name'),
+          description: this.$t('settingsSections.notifications.description')
+        },
+        {
+          icon: 'mdi-text-box',
+          name: this.$t('settingsSections.logs.name'),
+          description: this.$t('settingsSections.logs.description')
+        }
+      ]
+    };
+  },
+  async beforeMount() {
+    this.setAppBarOpacity({ opaqueAppBar: true });
+    this.setPageTitle({ title: this.$t('settings') });
+
+    if (this.$auth.user.Policy.IsAdministrator) {
+      this.systemInfo = (await this.$api.system.getSystemInfo()).data;
+    }
+  },
+  methods: {
+    ...mapActions('page', ['setPageTitle', 'setAppBarOpacity']),
+    isEmpty(object: any): boolean {
```

Comment:
```suggestion
    isEmpty(object: never): boolean {
```
Let's keep the happiness of the linter :)